src/core/public/saved_objects/saved_objects_client.ts

```diff
@@ -227,6 +234,11 @@ export class SavedObjectsClient {
     this.batchQueue = [];
   }
 
+  public setCurrentWorkspace(workspaceId: string): boolean {
+    this.currentWorkspaceId = workspaceId;
+    return true;
```

Comment:
Is `return true` necessary?

Reply:
For a standard `set` method, `return true` is not neeeded. Remove that.